comfyui_script_to_video_suite/s2v_nodes/s2v_executor_nodes.py

```
@@ -0,0 +1,96 @@
+# s2v_executor_nodes.py
+import re
+
+# Helper function to parse prompts from a section
+def _parse_prompts_from_section(section_text: str) -> list[str]:
+    matches = re.findall(r"PANEL\s+\d+:\s*(.*)", section_text, re.IGNORECASE)
+    return [match.strip() for match in matches]
+
+class PromptUnpacker:
+    """
+    Node #4a: Parses the full text from the PromptGenerator into clean, 
+    usable lists of prompts, ready for iteration.
+    """
+    @classmethod
+    def IS_CHANGED(cls, **kwargs):
+        return float("NaN")
+
+    @classmethod
+    def INPUT_TYPES(cls):
+        return { "required": { "prompt_text": ("STRING", {"multiline": True}), } }
+
+    RETURN_TYPES = ("PROMPTS_LIST", "PROMPTS_LIST", "STRING",)
+    RETURN_NAMES = ("image_prompts", "video_prompts", "meta_summary",)
+    FUNCTION = "unpack_prompts"
+    CATEGORY = "Script To Video Suite/Execution"
+
+    def unpack_prompts(self, prompt_text: str):
+        print("Executing 'Prompt Unpacker' node...")
+        if not prompt_text or not prompt_text.strip():
+            print("⚠️ WARNING: Input prompt_text is empty.")
+            return ([], [], "")
```

silent failure... may be add better error handling